components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -349,24 +341,18 @@ public sealed class PublisherJobService : IPublishServices<string> {
             List<PublishedDataSetVariableModel> variables = null;
             foreach (var writer in publishJob.WriterGroup.DataSetWriters) {
                 if (writer.DataSet.DataSetSource.Connection.IsSameAs(connection) &&
-                    writer.DataSetWriterId == dataSetWriterId) {
+                    writer.DataSetWriterName == dataSetWriterId) {
```

Comment:
`writer.DataSetWriterName == dataSetWriterId` is a bit confusing

Reply:
Will change the variable name.

Reply:
Updated